### PR TITLE
Added another reason to use CSS grids

### PR DESCRIPTION
**Pull request by Ethan-Master-Coding**
Added why CSS grids are better than HTML tables.

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

File: content/css/concepts/grids/grids.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-CSS Grid is a two-dimensional grid-based layout system, with rows, columns and gaps. This layout system makes it easier to design and fine-tune the layout of web pages without having to use positioning and floats.
+CSS Grid is a two-dimensional grid-based layout system, with rows, columns and gaps. This layout system makes it easier to design and fine-tune the layout of web pages without having to use positioning and floats. CSS grids are better becuase it has a lot more options to customisation compared to HTML tables.
```
Comment (yangc95) [inline content/css/concepts/grids/grids.md:16]:
```suggestion
CSS Grid is a two-dimensional grid-based layout system with rows, columns, and gaps. This layout system makes it easier to design and fine-tune the layout of web pages without having to use positioning and floats.
```

Agree with your point that when using CSS grid, there are lots of options for customization when deciding on the layout of a web page. However, I'm not sure if HTML tables is a great comparison for grid.
CSS Float is more comparable, but even then everyone has their preference for different reasons. I'd definitely look into this a bit more and the general discourse that exists on which parts of each technique are favored.

File: content/javascript/concepts/functions/functions.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Function declarations are used to create named functions. These functions can be
 The example code provided contains a function named `add()` that takes in 2 values and prints the sum of those numbers:
 
 ```js
-function add(number1, number2) {
+function sum(number1, number2) {
```
Comment (yangc95) [inline content/javascript/concepts/functions/functions.md:30]: I actually prefer this renaming as well 👍 let's make sure to change the corresponding explanation on line 27 to match our newly named function.

**Review by yangc95 — COMMENTED** [review]
Finished with the first review! Make sure to tag me once you've made the appropriate changes.

In addition, when you're going through all these entries and looking to make edits, it'd be great to take a look at the metadata at the top of the file. The descriptions shouldn't be too long and if you see any incomplete sentences, feel free to edit those as well. 
Ex: For the CSS entry on Grid, the trailing `html 1 2 3` can be removed.

**Comment by Ethan-Master-Coding** [comment]
@yangc95 I accepted those suggestions. They looked good.

**Review by Ethan-Master-Coding — COMMENTED** [review]
All edits look good and are correct.

File: content/javascript/concepts/functions/functions.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ Function declarations are used to create named functions. These functions can be
 The example code provided contains a function named `add()` that takes in 2 values and prints the sum of those numbers:
```
Comment (yangc95) [inline content/javascript/concepts/functions/functions.md:27]:
```suggestion
The example code provided contains a function named `sum()` that takes in 2 values and prints the sum of those numbers:
```

File: content/javascript/concepts/functions/functions.md (hunk 2 above)
Comment (yangc95) [reply content/javascript/concepts/functions/functions.md:30]: @Ethan-Master-Coding I've added a suggestion. Please take a look here.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Thanks for the corrections @Ethan-Master-Coding 

For reviewer #2: @KTom101 @Dusch4593
Wondering what you guys think about the comparison of CSS grids with HTML tables. Would you say this is a valid comparison or is there a better one that we should add to this intro?

I was thinking flexbox is a good comparison or keeping the intro info as it is now.

**Comment by Dusch4593** [comment]
> Thanks for the corrections @Ethan-Master-Coding
> 
> For reviewer2: @KTom101 @Dusch4593 Wondering what you guys think about the comparison of CSS grids and HTML tables. Would you say this is a valid comparison or is there a better one that we should add to this intro?
> 
> I was thinking flexbox is a good comparison or keeping the intro info as it is now.

I wouldn't be opposed to dedicating a sentence each to both HTML tables and Flexbox in the opening paragraph.

**Comment by Dusch4593** [comment]
Also, I'm going to delete the `functions.md` file since there seems to be a PR already made for it.

**Comment by Dusch4593** [comment]
Also, @Ethan-Master-Coding, for future PRs make sure to push changes from a branch *_besides main*_. This is encouraged so that we have a better time pull your changes down to our local machines if necessary. it's for this reason that this is one of the checkboxes on the PR template (which [you checked](https://github.com/Codecademy/docs/pull/461#issue-1077338958)).

**Comment by Ethan-Master-Coding** [comment]
@Dusch4593 When I did something with an edit after I made the PR it change the branch to main and I could not figure it out.

**Comment by Dusch4593** [comment]
@Ethan-Master-Coding We seem to have a merge conflict going on. Is your forked Docs repo's `main` branch (`Ethan-Master-Coding/docs:main`) caught up with the upstream `main` branch?(`Codecademy/docs:main`)

If not, could you please do so?

Screen Shot 2021-12-15 at 11 17 15 AM

**Comment by Dusch4593** [comment]
Things look clear to ahead and perform review 2. @Ethan-Master-Coding I'll let you know when I'm finished so you can take a look at any comments/suggestions that need resolved. After you're finished, let us know and we'll move onward. 😃

**Review by Dusch4593 — APPROVED** [review]
LGTM!

**Comment by Dusch4593** [comment]
@Ethan-Master-Coding, this looks about ready for merge. Before that, though, I made a few changes:

- Edited the meta-`Description` to include the comma you included in the opening paragraph.
- Small edits to `Description` and opening paragraph to include the point that Grid is a responsive layout system and alternative to positioning, floats, as well as Flexbox.
- Added sentence below CSS code snippet to explicitly mention `grid-item` being assigned as a `class` to each Grid item.

cc: @yangc95

**Comment by Dusch4593** [comment]
@Ethan-Master-Coding Changes are now live! https://www.codecademy.com/resources/docs/css/grids

Thank you again, Ethan, for another awesome contribution to Docs! 🎉 🎊

File: content/javascript/concepts/functions/functions.md (hunk 3 above)
Comment (Sagor2214) [reply content/javascript/concepts/functions/functions.md]: Help me

**Pull request merged**